### PR TITLE
Feature/financial resources integration

### PR DESCRIPTION
**Pull request by Bluesmile82**
This PR uses the financial resources real data to:

- Feed the Sankey and Bubble charts in support received
- Render different dropdowns and filter data depending on the tab (domestic, international, non-monetized)
- Feed the table in support needed: Adjust column width and fix search

Extra:
- Add scroll to top when clicking on the flagship links

Notes:
  - Non-monetized tab is disabled. We need more info on how to represent the data here.

image
image
image

### CODE REVIEW

**Comment by aabdaab** [comment]
@Bluesmile82 is it still WIP or ready to review?

**Comment by simaob** [comment]
@aabdaab and @santostiago can you review this PR please!=)

**Comment by aabdaab** [comment]
@Bluesmile82 I merged the PR in cw components so you can go ahead and fix the tooltip

**Comment by aabdaab** [comment]
image

Also, it looks like my sankey is missing something...?

**Comment by Bluesmile82** [comment]
I've created a PR in cw-components for the changes in the Sankey. Focus is fixed. The column height fixing is still in process (Will be done with more changes in cw-components in a different PR)

**Comment by Bluesmile82** [comment]
Ready again! Could you check this? I have to do another PR with some changes

File: app/javascript/app/components/flagship-buttons/flagship-buttons-component.jsx [hunk 1]
```diff
@@ -30,7 +30,10 @@ class FlagshipButtons extends PureComponent {
         key={s.name}
         flagshipImages={flagshipImages}
         section={s}
-        onClick={() => handleFilterChange(slug)}
+        onClick={() => {
+          if (scrollToTop) window.scrollTo(0, 0);
```
Comment (aabdaab) [inline app/javascript/app/components/flagship-buttons/flagship-buttons-component.jsx]: 👍 💯

**Pull request merged**